prov/efa/src/rdm/efa_rdm_rma.c

```diff
@@ -600,6 +600,10 @@ ssize_t efa_rdm_rma_writedata(struct fid_ep *ep, const void *buf, size_t len,
 	if (err)
 		return err;
 
+	err = efa_rdm_attempt_to_sync_memops(efa_rdm_ep, (void *)buf, desc);
```

Comment:
Hmmm this needs a backport. I am not sure how we can fix the earlier nccl 2.19 setmemops issue earlier without having this. fi_writedata is primarily called on p5. Probably we are lucky because plugin always call tsenddata first before calling this

Reply:
OK I take it back, it is called in writemsg anyway

Reply:
no aciton.